Sources/Logging/Protocols/LogDestination.swift

```diff
@@ -13,10 +13,12 @@ public protocol LogDestination: class {
     typealias ID = String
 
     var minLevel: Log.Level { get }
-    var formatter: LogItemFormatter { get }
     var id: ID { get }
 
-    func write(item: Log.Item, failure: @escaping (Error) -> ())
+    func write(item: Log.Item, failure: @escaping (Error) -> Void)
```

Comment:
💄 `onFailure`?

Reply:
As above, agreed 😄 

My thought process when naming this was from the perspective of `completion` / `failure`, but `onFailure` in a way names the "event" itself better.